includes/class-affiliatewp-leaderboard.php

```diff
@@ -230,26 +239,42 @@ public function affiliate_leaderboard( $atts, $content = null ) {
 	 */
 	public function show_leaderboard( $args = array() ) {
 
-		$defaults = apply_filters( 'affwp_leaderboard_defaults',
-			array(
-				'number'  => isset( $args['number'] ) ? $args['number'] : 10,
-				'orderby' => isset( $args['orderby'] ) ? $args['orderby'] : 'referrals',
+		/**
+		 * Filters the leaderboard shortcode attributes.
+		 *
+		 * @since 1.0
+		 *
+		 * @param array $attributes Shortcode attributes.
+		 *    $atts = [
+		 *      'earnings'  => (string) Toggle earnings visibility.
+		 *      'referrals' => (string) Toggle referrals visibility.
+		 *      'visits'    => (string) Toggle visits visibility.
+		 *      'number'    => (int)    Number of items to display.
+		 *      'orderby'   => (string) DB column to order results.
+		 *    ]
+		 */
+		$args = wp_parse_args(
+			$args,
+			apply_filters(
+				'affwp_leaderboard_defaults',
+				array(
+					'number'  => isset( $args['number'] ) ? $args['number'] : 10,
+					'orderby' => isset( $args['orderby'] ) ? $args['orderby'] : 'referrals',
+				)
 			)
 		);
 
-		$args = wp_parse_args( $args, $defaults );
-
 		// show an affiliate's earnings.
-		$show_earnings = isset( $args['earnings'] ) && ( 'yes' == $args['earnings'] || 'on' == $args['earnings'] ) ? true : false;
+		$show_earnings = isset( $args['earnings'] ) && ( 'yes' === $args['earnings'] || 'on' === $args['earnings'] ) ? true : false;
```

Comment:
```suggestion
		$show_earnings = isset( $args['earnings'] ) && in_array( $args['earnings'], array( 'on', 'yes' ), true ) ? true : false;
```

Just like my note above, you can also use `in_array`.